exe/doggy

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env ruby
 
+$LOAD_PATH.unshift(File.expand_path("../../lib", __FILE__))
```

Comment:
I think this is not necessary for a gem executable. `lib` folder gets added to the PATH anyways.